src/Integration.Vsix/Delegates/SonarAnalyzerWorkflowBase.cs

```diff
@@ -58,10 +59,24 @@ protected SonarAnalyzerWorkflowBase(Workspace workspace)
         }
 
         internal /* for testing purposes */ bool ShouldRegisterContextActionWithFallback(
-            IEnumerable<DiagnosticDescriptor> descriptors) =>
-            ShouldRegisterContextAction(descriptors)
-            // Fallback using SonarWay
-            ?? descriptors.Any(d => d.CustomTags.Contains(DiagnosticTagsHelper.SonarWayTag));
+            IEnumerable<DiagnosticDescriptor> descriptors)
+        {
+            // Let's disable all the utility analyzers under SonarLint
+            // - Note 1:
+            // There is already some logic within the analyzer to disable the execution when the output-cs/vbnet folder is not
+            // found which I decided to skip here.
+            // - Note 2:
+            // The CPD could be useful but is also disabled at the moment
+            if (descriptors.Any(d => d.Id.StartsWith("S9999-")))
+            {
+                Debug.Assert(descriptors.Count() == 1, "Not expecting to receive multiple utility-analyzer descriptors at once");
```

Comment:
The assertion message is potentially misleading - it assumes that all of the other descriptors are for utility analysers too. It might be better to say something like "Expecting utility-analysers descriptors to be passed on individually"  "without any other descriptors".